Tools/GNUMake/Make.defs

```diff
@@ -42,12 +42,12 @@ ifeq ($(USE_HIP),TRUE)
   DEBUG := FALSE  # Currently there is a compiler bug for DEBUG=TRUE.
   USE_CUDA := FALSE
   COMP := hip
-  HIP_PLATFORM = $(shell hipconfig --compiler)
-  ifeq ($(HIP_PLATFORM),nvcc)
-    $(error HIP_PLATFORM nvcc is not supported at this time. Use USE_CUDA to compile for NVIDIA platforms)
-  else ifeq ($(HIP_PLATFORM),hcc)
-    $(error HIP_PLATFORM=hcc is no longer supported)
-  else ifeq ($(HIP_PLATFORM),clang)
+  HIP_COMPILER = $(shell hipconfig --compiler)
+  ifeq ($(HIP_COMPILER),nvcc)
+    $(error HIP_COMPILER=nvcc is not supported at this time. Use USE_CUDA to compile for NVIDIA platforms)
+  else ifeq ($(HIP_COMPILER),hcc)
+    $(error HIP_COMPILER=hcc is no longer supported)
+  else ifeq ($(HIP_COMPILER),clang)
   else
     $(error Unknown HIP_PLATFORM=$(HIP_PLATFORM))
```

Comment:
This needs to be updated too.

Reply:
```Tools/GNUMake/comps/hip.mak:ifeq ($(HIP_PLATFORM),clang)
Tools/GNUMake/comps/hip.mak:else ifeq ($(HIP_PLATFORM),nvcc)
Tools/GNUMake/comps/hip.mak:  $(error HIP_PLATFORM nvcc is not supported at this time. Use USE_CUDA to compile for NVIDIA platforms.)
```

Reply:
Now it should be good to go

Reply:
Seems that there still is one place in Make.defs.

Reply:
`$(error Unknown HIP_PLATFORM=$(HIP_PLATFORM))` is still there, I think

Reply:
Can confirm:
```
Tools/GNUMake/Make.defs
52:    $(error Unknown HIP_PLATFORM=$(HIP_PLATFORM))
```